### PR TITLE
Defaults the save/rename dialog to the current preset name

### PR DESCRIPTION
**Pull request by randellhodges**
This defaults the store custom variable set dialog to the active visibility set name. Useful if you are constantly tweaking and saving the existing set.

### CODE REVIEW

**Comment by fieldOfView** [comment]
I like the idea, but it would seem to me that it only works correctly the first time; the active item is defined as a constant so it will not change when you select a different set to work on.

**Comment by randellhodges** [comment]
Actually, it does change. I'll admit to be a novice Cura/Qt/python developer, so I did a little copy/paste and not 100% sure if it is needed. I originally tried just doing a function call and not a property but it wouldn't work for me.

To test, I brought up the name/rename dialog and saw one name. Flipped to a different visibility set then brought up the name/rename dialog again and saw it reflected the change in visibility set.

**Comment by randellhodges** [comment]
Well, the one thing I didn't try before submitting the pull request is actually make a change to a setting visibility. Once you do that, Cura changes your active setting to "Custom selection".

Say you are editing a custom preset named "My Preset". If you make a visibility change, Cura changes your active preset to "Custom selection" so when you go to save it, you expect "My Preset" to be in the field name, but you are actually prompted with "Custom selection".

So, in a nutshell, this needs work, if it is even possible to determine what the preset was before Cura changed it for you.
